gcp_variant_transforms/libs/vcf_header_parser_test.py

```diff
@@ -78,42 +54,28 @@ def test_invalid_file(self):
     with temp_dir.TempDir() as tempdir:
       file_path = self._create_temp_vcf_file(lines, tempdir)
       try:
-        vcf_header_parser.get_merged_vcf_headers(file_path)
+        vcf_header_parser.get_vcf_headers(file_path)
         self.fail('Invalid VCF file must throw an exception.')
       except ValueError:
         pass
 
-  def test_incompatible_files(self):
-    # NS has Number=1 in file1, but Number=2 in file2.
-    file_1_lines = [
-        '##fileformat=VCFv4.2\n',
-        '##INFO=<ID=NS,Number=1,Type=Integer,Description="Number samples">\n',
-        '#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	Sample1	Sample2\n']
-    file_2_lines = [
-        '##fileformat=VCFv4.2\n',
-        '##INFO=<ID=NS,Number=2,Type=Integer,Description="Number samples">\n',
-        '##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">\n',
-        '#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	Sample3\n']
-
+  def test_empty_file(self):
+    lines = []
     with temp_dir.TempDir() as tempdir:
-      self._create_temp_vcf_file(file_1_lines, tempdir)
-      self._create_temp_vcf_file(file_2_lines, tempdir)
-      try:
-        vcf_header_parser.get_merged_vcf_headers(
-            os.path.join(tempdir.get_path(), '*.vcf'))
-        self.fail('Incompatible VCF files must throw an exception.')
-      except ValueError:
-        pass
+      file_path = self._create_temp_vcf_file(lines, tempdir)
+      header_fields = vcf_header_parser.get_vcf_headers(file_path)
+      self.assertItemsEqual([], header_fields.infos.keys())
+      self.assertItemsEqual([], header_fields.formats.keys())
 
   def test_gz(self):
     """Tests successfully parsing gz files."""
     file_path = testdata_util.get_full_file_path('valid-4.0.vcf.gz')
-    header_fields = vcf_header_parser.get_merged_vcf_headers(file_path)
+    header_fields = vcf_header_parser.get_vcf_headers(file_path)
     self.assertGreater(len(header_fields.infos), 1)
     self.assertGreater(len(header_fields.formats), 1)
 
   def test_non_existent_input_pattern(self):
     expected_header_fields = vcf_header_parser.HeaderFields({}, {})
     self.assertEqual(expected_header_fields,
-                     vcf_header_parser.get_merged_vcf_headers('randompath/*'))
+                     vcf_header_parser.get_vcf_headers('randompath/*'))
```

Comment:
Since we no longer use an input pattern, I think we should change this to something like `randompath.vcf`

Reply:
Done!